### DISCUSSION TITLE
Osu chat

### DISCUSSION THREAD

**Discussion by Rystars** [ppy/osu#17077] [Ideas]
https://user-images.githubusercontent.com/96302690/156580186-726ceacb-872f-4a82-bc77-aacd00f2a5fc.mp4

-Please make the osu chat tab visible when the keyboard exits

-Add keyboard suggestion

**Comment by bdach** [comment]
* We're aware that mobile keyboards don't work great with chat yet. It's not a high priority for now, desktop takes precedence. It'll be addressed when mobile takes a higher priority.
* What do you mean by "add keyboard suggestion"?

**Reply by Rystars** [reply]
![IMG_20220304_064538](https://user-images.githubusercontent.com/96302690/156672104-153cdad0-35e5-42ea-94cb-be211d73b6a0.jpg)

I mean the one I circled
